GetStartedDebugging/GetStartedDebugging/Program.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+class ArrayExample
+{
+    static void Main()
+    {
+        char[] letters = { 'f', 'r', 'e', 'd', ' ', 's', 'm', 'i', 't', 'h' };
+        string name = "";
+        int[] a = new int[10];
+        for (int i = 0; i < letters.Length; i++)
+        {
+            name += letters[i];
+            a[i] = i + 1;
+            SendMessage(name, a[i]);
+        }
+        Console.ReadKey();
+    }
+    static void SendMessage(string name, int msg)
+    {
+        Console.WriteLine("Hello, " + name + "! Count to " + msg);
+    }
+}
```

Comment:
This is your Lab02 stuff that should be in  your Lab02 repo, not here in your DSA.  Copy and paste it over, then delete this code out of here.